### PR TITLE
uplift/bot: add mozdata.ini for uplift/bot to specify user-agent

### PR DESCRIPTION
**Pull request by kevinluvian**
#1286 

This PR only add mozdata.ini to uplift/bot as requested by @calixteman to change the user-agent
previously, the config is replaced manually like:
https://github.com/mozilla/release-services/blob/bd3823b22243eb577300a8458e85a6dc75db10da/src/uplift/bot/uplift_bot/bugzilla.py#L39

as we don't want any regression issue, I only add user agent to the mozdata.ini, so the rest will still behave the same as before

### CODE REVIEW

File: src/uplift/bot/mozdata.ini [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[User-Agent]
+name = uplift-bot
```
Comment (La0) [inline src/uplift/bot/mozdata.ini]: I'm not sure that would work as expected in our Nix environment.
Could you add a unit test that makes a Bugzilla query, using [responses](https://github.com/getsentry/responses) to catch the query and checks the user agent in the headers ?

A good example in this project is [test_report.py](https://github.com/mozilla/release-services/blob/master/src/uplift/bot/tests/test_report.py). Please create a new file in the same folder (`test_libmozdata.py` ?)

Comment (kevinluvian) [reply src/uplift/bot/mozdata.ini]: Sure, will do that.
just now I only tested it using python shell, and `import config from libmozdata.bugzilla`
then I test to get the user agent from config using `config.get('User-Agent', 'name', 'libmozdata')` as shown here

https://github.com/mozilla/libmozdata/blob/319368f447624204a263697a772a07e3968b5339/libmozdata/connection.py#L54

**Comment by kevinluvian** [comment]
@La0 I have added a test for this, please take a look when you have time

**Comment by La0** [comment]
Thanks for your great work @kevinluvian !

**Pull request merged**